src/simput/index.js

```diff
@@ -60,5 +61,8 @@ function registerLocalEditors(Simput) {
   }
 }
 
-const Simput = (typeof window === 'undefined' ? {} : window).Simput;
-registerLocalEditors(Simput);
+if (typeof window !== 'undefined') {
+  // expose global so simput/types/vera/src/convert.js can use it.
+  window.Simput.InpHelper = InpHelper;
```

Comment:
I wouldn't put it under the Simput namespace.
Maybe Simput.type.vera.helper.InHelper?

Reply:
K, done